cdhweb/people/tests/conftest.py

```diff
@@ -0,0 +1,108 @@
+from datetime import date, timedelta
```

Comment:
can we just call this file "fixtures"?

Reply:
no, conftest is a special pytest name that gets loaded automatically

Reply:
oh! good to know